draft-ietf-quic-transport.md

```diff
@@ -124,10 +124,54 @@ provides:
 
 * Authenticated and encrypted header and payload
 
-QUIC uses UDP as a substrate to avoid requiring changes to legacy client
-operating systems and middleboxes.  QUIC authenticates all of its headers and
-encrypts most of the data it exchanges, including its signaling, to avoid
-incurring a dependency on middleboxes.
+QUIC establishes a connection, which is a stateful interaction between a client
+and server. The primary purpose of a connection is to support the exchange of
+data by an application protocol.
+
+Streams ({{streams}}) are the primary means by which an application protocol
```

Comment:
Saying 'primary' seems odd given there's no other option in v1.

You could say "Streams are the only means by which an application protocol exchange information in this version of QUIC."